src/reference/job-script-vars/var-list.txt

```diff
@@ -25,7 +25,9 @@ CYLC_WORKFLOW_RUN_DIR              # Location of the run directory in
 CYLC_WORKFLOW_HOST                 # Host running the workflow process
 CYLC_WORKFLOW_OWNER                # User ID running the workflow process
 
-CYLC_WORKFLOW_SHARE_DIR            # Workflow (or task!) shared directory (see below)
+CYLC_WORKFLOW_SHARE_DIR            # Workflow (or task!) shared directory
+CYLC_TASK_SHARE_CYCLE_DIR          # The share/cycle/<cycle point directory>
+                                   # for this task.
```

Comment:
Would be best if this was in this block, methinks

![image](https://github.com/user-attachments/assets/77fd326a-a909-4780-89ac-12dbf8ce1206)